crates/telio-nurse/src/qos.rs

```diff
@@ -388,13 +388,22 @@ impl Analytics {
     }
 
     /// Launch ping task for every node.
-    fn perform_ping(&mut self) {
+    async fn perform_ping(&mut self) {
```

Comment:
Why is this async now?

Reply:
This was a left over from a change I ended up removing 🤦 

Good catch, will remove.